rmw_implementation/src/functions.cpp

```diff
@@ -26,7 +26,7 @@
 #include "rcutils/format_string.h"
 #include "rcutils/types/string_array.h"
 
-#include "rcpputils/get_env.hpp"
+#include "rcpputils/env.hpp"
```

Comment:
This isn't right, as far as I can tell.  We deprecated `rcutils/get_env.h`, not `rcpputils/get_env.hpp`.  So I don't think this is going to work.

Reply:
Oh, never mind me.  I didn't read the whole description, the whole point of this is to add that :).